specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/preview/2022-01-15-privatepreview/AccessControlLists.json

```diff
@@ -0,0 +1,582 @@
+        }
+      ],
+      "properties": {
+        "addressFamily": {
+          "type": "string",
+          "description": "IP address family. Example: ipv4 | ipv6.",
+          "enum": [
+            "ipv4",
+            "ipv6"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "addressFamily"
+          }
+        },
+        "conditions": {
+          "type": "array",
+          "description": "Access Control List conditions.",
+          "x-ms-identifiers": [
+            "sequenceNumber"
+          ],
+          "items": {
+            "description": "Access Control List condition model.",
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "./common.json#/definitions/AnnotationResource",
+                "description": "Common properties for annotated type."
+              }
+            ],
+            "properties": {
+              "sequenceNumber": {
+                "type": "integer",
+                "format": "int32",
+                "description": "sequenceNumber of the Access Control List."
+              },
+              "action": {
+                "type": "string",
+                "description": "action. Example: allow | deny.",
+                "enum": [
+                  "allow",
+                  "deny"
+                ],
+                "x-ms-enum": {
+                  "modelAsString": true,
+                  "name": "conditionActionType"
+                }
+              },
+              "destinationAddress": {
+                "type": "string",
+                "description": "destinationAddress. Example: any | 1.1.1.0/24 | 1.1.10.10"
+              },
+              "destinationPort": {
+                "type": "string",
+                "description": "destinationPort. Example: any | 1253"
+              },
+              "sourceAddress": {
+                "type": "string",
+                "description": "sourceAddress. Example: any | 1.1.1.0/24 | 1.1.10.10"
+              },
+              "sourcePort": {
+                "type": "string",
+                "description": "sourcePort. Example: any | 1253"
+              },
+              "protocol": {
+                "type": "integer",
+                "format": "int32",
+                "description": "TCP/IP protocol as defined in the list of IP protocol numbers. Example: 255 (any) | 0 | 1."
+              }
+            },
+            "required": [
+              "sequenceNumber",
+              "action",
+              "destinationAddress",
+              "destinationPort",
+              "sourceAddress",
+              "sourcePort",
+              "protocol"
+            ]
+          }
+        }
+      },
+      "description": "AccessControlListPatchProperties define the patchable resource properties."
+    },
+    "AccessControlListPatch": {
+      "type": "object",
+      "x-ms-azure-resource": true,
```

Comment:
>"x-ms-azure-resource": true, [](http://example.com/codeflow?start=6&length=28)

Patch input not necessarily have all the resource properties, so "x-ms-azure-resource" might not valid here.